CONTRIBUTING.rst

```diff
@@ -13,42 +13,28 @@ TL;DR
 
 .. code-block:: console
 
-    $ # Export libpq env vars for PG connection
-    $ export PGDATABASE=procrastinate PGHOST=localhost PGUSER=postgres
-
-    $ # Launch PostgreSQL within Docker
+    $ # Launch services within Docker
     $ docker-compose up -d
 
-    $ # Explore tox entrypoints
-    $ tox -l
+    $ # Create procrastinate databases
+    $ docker-compose run --rm procrastinate migrate
+
+    $ # Launch demo
```

Comment:
Why "Launch demo" here? I'd say "Display the procrastinate command's help message" or something…